sky/services/platform/host_app.mojom

```diff
@@ -0,0 +1,20 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module flutter.platform;
+
+/// A service provided by the host application that can receive
+/// application-specific messages sent by the Flutter application.
+[ServiceName="flutter::platform::HostApplication"]
+interface HostApplication {
+  SendToHost(string message) => (string? reply);
+};
+
+/// A service provided by the Flutter framework that can receive messages
+/// sent by the host.
+[ServiceName="flutter::platform::EmbeddedApplication"]
+interface EmbeddedApplication {
+  SendToFlutter(string message) => (string? reply);
+};
```

Comment:
Is there a reason to have two different interfaces?  Why not just have one?  You could then call "SendToHost" and "SendToFlutter" just "SendString"